lib/sidekiq/web/application.rb

```diff
@@ -77,9 +77,11 @@ def initialize(inst)
       get "/metrics/:name" do
         @name = route_params(:name)
         @period = h(url_params("period") || "1h")
-        q = Sidekiq::Metrics::Query.new
-        @periods = METRICS_PERIODS
+        # Periods larger than 8 hours are not supported for histogram chart
+        @period = "8h" if @period.to_i > 8
+        @periods = METRICS_PERIODS.reject { |k, v| k.to_i > 8 }
```

Comment:
Here's this constraint in action. Without it, you'd encounter an error when viewing 24h metrics and linking to the job metrics.


https://github.com/user-attachments/assets/c5b074b5-8ac4-49a5-812a-48893c98c44d